vite.config.js

```diff
@@ -20,4 +20,9 @@ export default defineConfig({
             '~bootstrap': path.resolve(__dirname, 'node_modules/bootstrap'),
         }
     },
+    server: {
+        hmr: {
+            host: 'localhost',
```

Comment:
since this is part of the HMR config, this shouldn't mess with anything in prod right?

Reply:
Shouldn't do, it's just a WSL2 thing